docs/section18/Rule_18-1

```diff
@@ -0,0 +1,72 @@
+# HVAC_SystemZoneAssignment - Rule 18-1  
+**Schema Version:** 0.0.22  
+
+**Rule ID:** 18-1  
+**Rule Description:** Buildings in CZ 0-3A with predominant HVAC BAT = public assembly, <120,000 ft2 have baseline HVAC system type 4 (PSZ HP)
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 18 HVAC_SystemZoneAssignment  
+**Appendix G Section Reference:** Table G3.1.1.3
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. Buildings in CZ 0-3A
+2. predominant HVAC BAT = public assembly
+3. building area < 120,0000 ft2
+4. the zone does not meet one if the G3.1.1.1 exceptions
+
+**Manual Check:** Yes  
+**Evaluation Context:** Evaluate whether each Zone is served by the correct HVAC system type  
+**Data Lookup:**   
+**Function Call:** 
+
+1. zone_meets_any_G3_1_1_exception(P-RMR,B-RMR,zone)
+2. get_baseline_system_types()
+3. get_HVAC_building_area_types_and_zones
+3. get_baseline_system_types
+
+**Applicability Checks:**
+
+1. Check if the climate zone is CZ0-3A: `ASHRAE229.weather.climate_zone in ["CZ0A","CZ0B","CZ1A","CZ1B","CZ2A","CZ2B","CZ3A"]`
+	- set weather_applicable = TRUE: `weather_applicable = TRUE`
+	
+2. get the list of HVAC systems and zones: `list_of_systems_and_zones = get_HVAC_building_area_types_and_zones(B-RMR)`
+	- find the predominant HVAC BAT by first sorting the list by area: `sorted_list = sorted(list_of_systems_and_zones.items(), key=lambda x: x[1]["AREA"], reverse = True)`
+	- the predominant building area type will be the first item in the sorted list: `if sorted_list[0][0] == "PUBLIC_ASSEMBLY":`
+		- set area_type_applicable to true: `area_type_applicable = TRUE`
+		
+3. find the building area and check whether it's less than 120,000 ft2.  Start by creating an area variable: `building_area = 0`
+	- We can use the list_of_systems_and_zones to get the total building floor area.  Start by looping through the list: `for x in list_of_systems_and_zones:`
+		- x will be the various HVAC BAT.  list_of_systems_and_zones[x]["AREA"] will give the total area of this BAT: `building_area = building_area + list_of_systems_and_zones[x]["AREA"]
+	- check if building_area < 120000: `if building_area < 120000:`
+		- set area_applicable to true: `area_applicable = TRUE` 
```

Comment:
We agreed that HVAC system type is set based on the total building area, not just the area for the given BAT, right?